docker/release/Dockerfile

```diff
@@ -2,15 +2,19 @@ FROM ubuntu:16.04
 
 ENV IROHA_HOME /opt/iroha
 ENV IROHA_BUILD /opt/iroha/build
+ARG node
 
 COPY iroha.deb /tmp/iroha.deb
+COPY entrypoint.sh /entrypoint.sh
```

Comment:
Move these `COPY`s below, where it needed

First - exactly before the running gdebi
second - exactly before `ENTRYPOINT`

Reply:
fixed